src/styles/components/dropdown-panels/styles.less

```diff
@@ -0,0 +1,36 @@
+& when (@dropdown-panels-enabled) {
+
+  .dropdown-panel-group {
+    position: relative;
+
+    > .dropdown-panel {
+      box-shadow: @modal-shadow;
+      max-width: 100%;
+      position: absolute;
+      right: 0;
+      z-index: @z-index-overlay;
+
+      &.dropdown-panel-full-width {
+        width: 100%;
+      }
+
+      &.dropdown-panel-left {
+        left: 0;
+        right: auto;
+      }
+
+      &.dropdown-panel-animated {
+        margin-top: -10px;
```

Comment:
The `Dropdown` currently uses a hard-coded value of `30px`, maybe we should use this here? And I do think it would be nice to make use of `transform` here, as addressed here: https://github.com/dcos/dcos-ui/pull/1458/files#r90519960